server/manifest.js

```diff
@@ -51,7 +51,7 @@ const manifest = module.exports = {
 
 };
 
-if (process.env.NODE_ENV === 'dev') {
+if (process.env.NODE_ENV !== 'prod') {
```

Comment:
`prod` vs. `production`?  I know the latter is more widely used in the node world– for example, that's the only time when waterline would protect you from using nasty, nasty migration strategies.